src/CoreBundle/ResourceCollection/ResourceCollectionInformationServiceInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace ChameleonSystem\CoreBundle\ResourceCollection;
+
+interface ResourceCollectionInformationServiceInterface
+{
+    /**
+     * Returns true if the resource collection is enabled and not locked or otherwise
+     * blocked for the current request.
+     */
+    public function isActive(): bool;
```

Comment:
Is this used somewhere already?

Reply:
Yes - in templates that load resources into the header or the footer depending on the resource collection being active.

Reply:
But wouldn't I need to see these calls to isActive() in this PR. Because they have to be new (?)

Reply:
no - because I am using them in a project - not in the core.